pymc3/gp/gp.py

```diff
@@ -418,15 +417,15 @@ def marginal_likelihood(self, name, X, y, noise, is_observed=True, **kwargs):
         if not isinstance(noise, Covariance):
             noise = pm.gp.cov.WhiteNoise(noise)
         mu, cov = self._build_marginal_likelihood(X, noise)
-        chol = cholesky(stabilize(cov))
+        cov = stabilize(cov)
```

Comment:
I'm not sure the stabilisation is needed here since noise is already being added to the diagonal in `_build_marginal_likelihood`.